catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/StdidcAttribute.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.alliance.transformer.nitf.common;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.function.Function;
+
+import org.codice.alliance.transformer.nitf.ExtNitfUtility;
+import org.codice.imaging.nitf.core.tre.Tre;
+
+import ddf.catalog.data.AttributeType;
+import ddf.catalog.data.impl.BasicTypes;
+import ddf.catalog.data.types.Location;
+
+/**
+ * TRE for "STDIDC "Standard ID"
+ */
+public class StdidcAttribute extends NitfAttributeImpl<Tre> {
+    private static final List<NitfAttribute<Tre>> ATTRIBUTES = new LinkedList<>();
```

Comment:
❓ Why a LinkedList?

Reply:
Copied from another ntif attribute class. Changed to ArrayList.